FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,25 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/f0556793b45bd971a01f1d6b9fd707367304df54/FlashEnglish/Sources/Views/Home/HomeView.swift#L13"></span>Limit vertical whitespace to a single empty line; currently 3<br />
<code>vertical_whitespace</code> <code>HomeView.swift:13</code></td>
    </tr>
  </tbody>
</table>